prisma/users.ts

```diff
@@ -0,0 +1,307 @@
+        id: "407351772575694920",
+    },
+    {
+        id: "407351772575694921",
+    },
+    {
+        id: "407351772575694922",
+    },
+    {
+        id: "407351772575694923",
+    },
+    {
+        id: "407351772575694924",
+    },
+    {
+        id: "407351772575694925",
+    },
+    {
+        id: "407351772575694926",
+    },
+    {
+        id: "407351772575694927",
+    },
+    {
+        id: "407351772575694928",
+    },
+    {
+        id: "407351772575694929",
+    },
+    {
+        id: "407351772575694930",
+    },
+    {
+        id: "407351772575694931",
+    },
+    {
+        id: "407351772575694932",
+    },
+    {
+        id: "407351772575694933",
+    },
+    {
+        id: "407351772575694934",
+    },
+    {
+        id: "407351772575694935",
+    },
+    {
+        id: "407351772575694936",
+    },
+    {
+        id: "407351772575694937",
+    },
+    {
+        id: "407351772575694938",
+    },
+    {
+        id: "407351772575694939",
+    },
+    {
+        id: "407351772575694940",
+    },
+    {
+        id: "407351772575694941",
+    },
+    {
+        id: "407351772575694942",
+    },
+    {
+        id: "407351772575694943",
+    },
+    {
+        id: "407351772575694944",
+    },
+    {
+        id: "407351772575694945",
+    },
+    {
+        id: "407351772575694946",
+    },
+    {
+        id: "407351772575694947",
+    },
+    {
+        id: "407351772575694948",
+    },
+    {
+        id: "407351772575694949",
+    },
+    {
+        id: "407351772575694950",
+    },
+    {
+        id: "407351772575694951",
+    },
+    {
+        id: "407351772575694952",
+    },
+    {
+        id: "407351772575694953",
+    },
+    {
+        id: "407351772575694954",
+    },
+    {
+        id: "407351772575694955",
+    },
+    {
+        id: "407351772575694956",
+    },
+    {
+        id: "407351772575694957",
+    },
+    {
+        id: "407351772575694958",
+    },
+    {
+        id: "407351772575694959",
+    },
+    {
+        id: "407351772575694960",
+    },
+    {
+        id: "407351772575694961",
+    },
+    {
+        id: "407351772575694962",
+    },
+    {
+        id: "407351772575694963",
+    },
+    {
+        id: "407351772575694964",
+    },
+    {
+        id: "407351772575694965",
+    },
+    {
+        id: "407351772575694966",
+    },
+    {
+        id: "407351772575694967",
+    },
+    {
+        id: "407351772575694968",
+    },
+    {
+        id: "407351772575694969",
+    },
+    {
+        id: "407351772575694970",
+    },
+    {
+        id: "407351772575694971",
+    },
+    {
+        id: "407351772575694972",
+    },
+    {
+        id: "407351772575694973",
+    },
+    {
+        id: "407351772575694974",
+    },
+    {
+        id: "407351772575694975",
+    },
+    {
+        id: "407351772575694976",
+    },
+    {
+        id: "407351772575694977",
+    },
+    {
+        id: "407351772575694978",
+    },
+    {
+        id: "407351772575694979",
+    },
+];
```

Comment:
why so many doe

Reply:
for the 2048 leaderboard paging. could cut it to 25 to get 2 pages. idk